Crawler.py

```diff
@@ -27,6 +29,8 @@ class Crawler:
     def run(self):
         # Reading csv file
         df = pd.read_csv('data/changes.csv', index_col='Event')
+        #Creating notifier
+        Notfiy = Notifier()
```

Comment:
Typo in `Notify`

Reply:
I would also call it `notifier`, to avoid confusion with the method `notify()`.
Lowercase for variables.